app/src/main/kotlin/com/hedvig/app/feature/offer/OfferViewModel.kt

```diff
@@ -145,7 +140,6 @@ abstract class OfferViewModel : ViewModel() {
 
   abstract fun onOpenCheckout()
   abstract fun reload()
-  abstract fun onDiscardOffer()
```

Comment:
Another case where the round trip to VM wasn't needed. Discarding the offer simply means going back to the main activity, so view does that directly.